src/browser/webview/mattermost.js

```diff
@@ -118,11 +118,13 @@ notification.override({
         currentWindow.focus();
       }
       else {
-        currentWindow.show();
+        if (mainWindow.isMinimized()) mainWindow.restore()
```

Comment:
Should use `currentWindow`. Probably should use `else if` because of nested `if` statement.